api/loginApiClient.module.ts

```diff
@@ -1,27 +1,44 @@
 ﻿const API_URL = "https://testswitch-api-staging.herokuapp.com/candidates"; //TODO change to /signin when endpoint exists
 
-async function loginCredentialsIncorrect(): Promise<boolean>{
-    const response = await fetch(API_URL);
-    return response.status == 200;
-
-    //TODO add if(response.status == 403) then not authorized, anything else is another issue.
-}
-
-export async function apiTryLogin(userId: string, password: string): Promise<boolean> {
+function clientSideValidation(userId: string, password: string): boolean {
```

Comment:
What about calling this `allCredentialsProvided()`? That says a bit more about what validation it's doing.
I think this belongs on the component instead of the API client as well. We should only be implementing the validation this way if the user is loging in through our login form.